doc/guide/Guide_Euler.ipynb

```diff
@@ -190,6 +190,22 @@
     "and the Wiki https://scicomp.ethz.ch/wiki/Transition_from_LSF_to_Slurm for a mapping of `lsf` commands to their `slurm` equivalents."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 7. Optional: install Climada Petals\n",
+    "\n",
+    "To install Climada Petals, repeat the steps described in step 4 as option A, but replacing the climada_python with the climada_petals repository:\n",
+    "```bash\n",
+    "cd /cluster/project/climate/$USER  # or wherever you plan to download the repository\n",
+    "git clone https://github.com/CLIMADA-project/climada_petals.git  # unless this has been done before\n",
+    "cd climada_petals\n",
+    "git checkout develop\n",
```

Comment:
```suggestion
```

This step is not part of the instructions for Climada Core in step 4. I think we can leave it out and instead suggest in the text that users are free to check out other branches? 🤔